Apps/Admin/Client/Store/Communications/CommunicationsEffects.cs

```diff
@@ -112,7 +112,7 @@ public async Task HandleUpdateAction(CommunicationsActions.UpdateAction action,
         if (response.IsSuccessStatusCode && response.Content != null && response.Content.ResultStatus == ResultType.Success)
```

Comment:
Do we need to ApiResponse? or just try and catch?

Reply:
Hmm, I'll have to think about that.  There is very little difference functionally.